tests/jerry/generators/generate-arguments-2.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+
+file = open("tests/jerry/arguments-2.js", "w")
+
+random.seed(87673)
+
+file.write("function f_arg (arguments)\n{\n    return arguments;\n};\n")
+file.write("function f (a, b, c)\n{\n    return arguments;\n};\n")
+
+for i in range(300):
+
+    N1 = random.randint(-99999999999, 99999999999)
+    N2 = random.randint(-99999999999, 99999999999)
+    N3 = random.randint(-99999999999, 99999999999)
+
+    file.write("assert (f_arg (" + str(N1) + ") === (" + str(N1) + "));\n")
+    file.write("assert (f_arg (" + str(N1) + ") !== (" + str(int (N1 - 1)) + "));\n")
+    file.write("assert (f_arg (" + str(N1) + ") !== (" + str(int(N1 + 1)) + "));\n")
+
+for i in range(400):
+
+    N1 = random.randint(-99999999999, 99999999999)
+    N2 = random.randint(-99999999999, 99999999999)
+    N3 = random.randint(-99999999999, 99999999999)
+    N4 = random.randint(-99999999999, 99999999999)
+    N5 = random.randint(-99999999999, 99999999999)
+    N6 = random.randint(-99999999999, 99999999999)
+    while N6 <= 4 and N6 >=0:
+        N6 = random.randint(-99999999999, 99999999999)
```

Comment:
Please do not use a while here. Maybe it you can write something like:
```python
if random.randint(0, 1):
  N6 = random.randint(5, 99999999999)
else:
  N6 = random.randint(-99999999999, -1)
```

Reply:
Solved